<!doctype html><html lang=en-us><head><title>OMSF</title><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta name=description content="The Open Molecular Software Foundation"><meta name=author content><meta property="og:title" content="Careers"><meta property="og:description" content="About OMSF
The Open Molecular Software Foundation (OMSF) is a US-based 501(c)(3) nonprofit organization focused on facilitating collaboration and software development for molecular sciences. OMSF aims to create a cost-effective organization to support an ecosystem of open source projects in molecular sciences, build expertise around managing and governing these projects, explore pathways to sustainability, and accelerate innovation by sharing our knowledge and tools under open licenses.
All OMSF positions are currently remote and we generally accept applications worldwide, provided that the OMSF will be able to enter in an appropriate working agreement with the applicant in their designated country of residence."><meta property="og:type" content="article"><meta property="og:url" content="https://omsf.io/about/careers/"><meta property="article:section" content="about"><meta property="article:published_time" content="2022-11-17T13:17:20+02:00"><meta property="article:modified_time" content="2022-11-17T13:17:20+02:00"><meta name=twitter:card content="summary"><meta name=twitter:title content="Careers"><meta name=twitter:description content="About OMSF
The Open Molecular Software Foundation (OMSF) is a US-based 501(c)(3) nonprofit organization focused on facilitating collaboration and software development for molecular sciences. OMSF aims to create a cost-effective organization to support an ecosystem of open source projects in molecular sciences, build expertise around managing and governing these projects, explore pathways to sustainability, and accelerate innovation by sharing our knowledge and tools under open licenses.
All OMSF positions are currently remote and we generally accept applications worldwide, provided that the OMSF will be able to enter in an appropriate working agreement with the applicant in their designated country of residence."><meta name=generator content="Hugo 0.85.0"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin=anonymous><link rel=stylesheet href=https://omsf.io/fontawesome/css/all.min.css><link rel=preconnect href=https://fonts.googleapis.com><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link href="https://fonts.googleapis.com/css2?family=Barlow:wght@400;600;700;900&display=swap" rel=stylesheet><link rel=stylesheet type=text/css href=/css/style.css><link rel=apple-touch-icon sizes=180x180 href=/apple-touch-icon.png><link rel=icon type=image/png sizes=32x32 href=/favicon-32x32.png><link rel=icon type=image/png sizes=16x16 href=/favicon-16x16.png><link rel=manifest href=/site.webmanifest><meta name=msapplication-TileColor content="#da532c"><meta name=theme-color content="#ffffff"><script type=text/javascript>var _paq=window._paq=window._paq||[];_paq.push(['trackPageView']),_paq.push(['enableLinkTracking']),function(){var d="https://omsf.matomo.cloud/",b,a,c;_paq.push(['setTrackerUrl',d+'matomo.php']),_paq.push(['setSiteId','1']),b=document,a=b.createElement('script'),c=b.getElementsByTagName('script')[0],a.type='text/javascript',a.async=!0,a.src='//cdn.matomo.cloud/omsf.matomo.cloud/matomo.js',c.parentNode.insertBefore(a,c)}()</script></head><body><div id=container><header><a class=logo href=/><img src=/images/logo-web.png alt=OMSF></a><nav><ul><li><a class=active href=/about/mission>About</a></li><li><a href=/services/fiscal-sponsorship/>Services</a></li><li><a href=/projects/host-project>Projects</a></li><li><a href=https://news.omsf.io>News</a></li><li><a href=/support/how-to>Support</a></li></ul></nav><button onclick=myFunction() id=toggle-main-menu-mobile class="hamburger hamburger--slider" type=button>
<span class=hamburger-box><span class=hamburger-inner></span></span></button><div id=mobile-submenu style=display:none class=mobile-submenu><a class=active href=/about/mission>About</a>
<a href=/services/fiscal-sponsorship/>Services</a>
<a href=/projects/host-project>Projects</a>
<a href=https://news.omsf.io>News</a>
<a href=/support/how-to>Support</a></div><script>function myFunction(){var a=document.getElementById("mobile-submenu");a.style.display==="none"?a.style.display="block":a.style.display="none"}</script></header><main><div class=sub-wrapper><div class=sub><a href=/about/mission/><span>Mission</span></a>
<a href=/about/people/><span>People</span></a>
<a class=active href=/about/careers/><span>Careers</span></a>
<a href=/about/legal/><span>Documents</span></a></div></div><article><h1>Careers</h1><p><strong>About OMSF</strong></p><p>The Open Molecular Software Foundation (OMSF) is a US-based 501(c)(3) nonprofit organization focused on facilitating collaboration and software development for molecular sciences. OMSF aims to create a cost-effective organization to support an ecosystem of open source projects in molecular sciences, build expertise around managing and governing these projects, explore pathways to sustainability, and accelerate innovation by sharing our knowledge and tools under open licenses.</p><p>All OMSF positions are currently remote and we generally accept applications worldwide, provided that the OMSF will be able to enter in an appropriate working agreement with the applicant in their designated country of residence. Some positions may be restricted to the US so please make sure to read the full job description before applying. Please note that you will need to have a valid work permit, as OMSF has a limited ability to sponsor any relocation visas at this stage. Please reach out to discuss visa options, if needed. Depending on your location, you might occasionally need to participate in meetings outside of normal business hours due to time zone differences.</p><p>When applying, please use the application forms linked below. Please reach out to <a href=mailto:careers@omsf.io>careers@omsf.io</a> if you have any additional questions or need help.</p><div class=header-two><img src=/OMSF/images/H2before.png alt><h2 id=openadmet-science-lead--machine-learning-models-and-active-learning>OpenADMET Science Lead â€“ Machine Learning Models and Active Learning</h2></div><p><strong>Job Description</strong>
The Open Molecular Software Foundation is seeking a Science Lead for the OpenADMET project, specializing in Machine Learning Models and Active Learning Strategies. The Science Lead will be responsible for overseeing the selection, implementation, and refinement of machine learning models and active learning strategies to drive the development of open source ADMET (Absorption, Distribution, Metabolism, Excretion, and Toxicity) models built from both public datasets and newly generated open ADMET datasets generated by the collaborative OpenADMET project. This position supervises and collaborates with scientific roles on the OMSF OpenADMET team, collaborates closely with the OpenADMET technical team to aid the development of robust training and inference infrastructure, and provides domain expertise for optimizing machine learning model development and deployment. This role is also instrumental in recruiting and liaising with a Technical Advisory Committee consisting of experts from academia and industry to advise on model strategies that maximize the utility of new open ADMET data generated by the project.</p><p>The Science Lead will work remotely in collaboration with diverse teams, primarily based in Europe, Australia, and the United States, and may require occasional travel. This fully remote, grant-funded position has a minimum duration of 2 years, with potential for extension based on available funding. The Science Lead will report directly to the OpenADMET Project Director. Learn more about OpenADMET <a href=https://openadmet.org/><strong>here</strong></a>.</p><p><strong>Key Responsibilities</strong>
<em>Machine Learning Model Development and Oversight</em>:</p><ul><li>Lead, manage, and contribute to efforts to automate training, assessment, and deployment of high-quality ligand-based (2D) ADMET models, ensuring state-of-the-art predictive capabilities.</li><li>Oversee the implementation of advanced structure-based (3D) ADMET models to leverage protein-ligand structural data, enabling accurate prediction of ADMET properties and enhancing generalizability across species and human genetic variations. This will involve collaboration with the OMSF OpenFold team (OpenFold.io).</li><li>Develop multitask models, uncertainty prediction approaches (e.g., model ensembling, Gaussian Process regression), and mechanistic models to optimize predictive performance and data utilization.</li><li>Drive the integration of new machine learning architectures and active learning strategies, leveraging recent innovations and literature as well as blind community challenges to ensure models remain at the cutting edge of accuracy and reliability.
<em>Active Learning and Data Curation</em></li><li>Establish and manage a continuous, automated active learning service to prioritize compounds for ADMET assays and structural biology, guiding data acquisition efforts for optimal model enhancement.</li><li>Develop strategies for rapid model re-training to incorporate new data, maximizing model accuracy on chemical spaces of interest while maintaining cost-efficiency.</li><li>Collaborate with internal and external stakeholders to refine active learning pipelines, ensuring model feedback loops are effectively managed.
<em>Scientific Leadership and Collaboration</em></li><li>Supervise scientific roles on the OpenADMET team, providing guidance on machine learning model design, data management, and active learning best practices.</li><li>Work closely with the technical team to ensure infrastructure supports efficient model training, deployment, validation, and active learning cycles.</li><li>Work closely with experimental groups</li><li>Engage with the Technical Advisory Committee and other experts to refine project strategies, keeping OpenADMETâ€™s machine learning approaches aligned with cutting-edge practices.</li><li>Engage with the Industry Advisory Board to integrate feedback on model performance, impact, and utility from pharma, biotech, and software vendors using OpenADMET models
<em>Documentation and Dissemination</em></li><li>Document best practices in a &ldquo;Living Best Practices Review&rdquo; and other OMSF publications, contributing to a foundation for reproducible, scalable machine learning in drug discovery.</li><li>Maintain and expand the software documentation for OpenADMET models and active learning processes, supporting both internal teams and external collaborators.</li></ul><p><strong>Qualifications</strong></p><ul><li>Advanced degree in a relevant field such as data science, ML/AI, bioinformatics, or a related discipline (PhD preferred, but candidates with a Masterâ€™s degree and significant experience are welcome).</li><li>Proven expertise in machine learning model development for small molecules properties, particularly for 2D ADMET applications or drug discovery.</li><li>Strong background in active learning methodologies, model uncertainty assessment, and model deployment for scientific research.</li><li>Familiarity with FAIR data principles, and experience in building reproducible data and model repositories.</li><li>Experience leading scientific teams or managing multi-faceted research projects, ideally with interdisciplinary and remote collaboration.</li><li>Excellent communication skills with the ability to engage cross-functional teams and advisory committees.
<em>Preferred Qualifications</em></li><li>Technical expertise in Python and machine learning frameworks (e.g., PyTorch, JAX, or other deep learning framework).</li><li>Experience with cloud computing for machine learning (e.g., AWS SageMaker, AWS Batch, modal, lambda, anyscale, GCP Vertex, Azure ML).</li><li>Experience with molecular data processing and modeling tools, such as RDKit, DeepChem, or similar.</li><li>Understanding of the drug discovery lifecycle and the role of machine learning in drug discovery.</li><li>Ability to understand and tune machine learning model performance.</li><li>Familiarity with continuous integration and automated deployment tools (e.g., github actions, conda-forge, CI/CD for model pipelines).</li><li>Knowledge of containerization and orchestration tools like Docker and Kubernetes for scalable model deployment.</li><li>Experience contributing to or leading open source projects and publications in computational molecular sciences or related fields.</li><li>Proven expertise in machine learning model development for small molecules properties, using 3D representations for ADMET applications or drug discovery.</li><li>Comfortable working in a distributed global team.
<strong>Compensation</strong>
This is a full-time, fixed contract position with an expected gross salary range of $150,000-$165,000, depending on qualifications, experience, and location. OMSF provides standard benefits, including healthcare, retirement contributions, paid time off, and other employer contributions per local regulations. Salary will be negotiated in the local currency, varying by location due to differences in mandatory employer contributions.
<strong>Location</strong>
OMSF is a fully remote organization. For this role, we will accept only applications from candidates who are based in the US due to federal funding restrictions.
<strong>How to Apply</strong>
Please apply using this <a href=https://forms.gle/g2uWWzx86a9xAYWF8><strong>form</strong></a>.</li></ul><p><div class=header-two><img src=/OMSF/images/H2before.png alt><h2 id=openadmet-senior-scientist--data-curation-and-pipeline-integration>OpenADMET Senior Scientist â€“ Data Curation and Pipeline Integration</h2></div><strong>Job Description</strong>
The Open Molecular Software Foundation is seeking a Senior Scientist specializing in Data Curation and Pipeline Integration for the OpenADMET project. This role focuses on ensuring the accuracy, quality, and accessibility of ADMET data used for machine learning applications, while developing and optimizing data workflows for seamless data ingestion, processing, and dissemination. The Senior Scientist will collaborate with experimental scientists at Octant/UCSF and the OpenADMET team to establish best practices for data curation and workflow integration. This includes preparing data in machine learning ready formats and managing data deposition into FAIR (Findable, Accessible, Interoperable, Reusable) repositories and ensuring the quality and reliability of all ADMET dataset classes.</p><p>This fully remote, grant-funded position has a minimum duration of 2 years, with potential for extension depending on available funding. Due to time zone differences, occasional meetings outside standard business hours may be necessary. Our teams are primarily based in Europe, Australia, and the United States, and some travel may occasionally be required.</p><p>The Senior Scientist â€“ Data Curation and Pipeline Integration will report directly to the OpenADMET project lead, while closely collaborating with the OpenADMET ML Infrastructure team, project stakeholders, and external partners. This role is ideal for a resourceful individual eager to shape data-driven projects that advance ADMET research in a collaborative, open-source environment. Learn more about OpenADMET <a href=https://openadmet.org/><strong>here</strong></a>.</p><p><strong>Key Responsibilities</strong>
<em>Data Curation and Quality Assurance</em></p><ul><li>Manage, curate, and maintain high-quality ADMET datasets in alignment with FAIR principles, ensuring accuracy and accessibility for machine learning applications.</li><li>Collaborate with experimental scientists to establish and document best practices for data curation, validation, and dissemination.</li><li>Develop quality checks and validation mechanisms to ensure data reliability across various ADMET dataset classes.
Pipeline Development and Optimization*</li><li>Design, implement, and optimize data ingestion and processing workflows using coding best practices (e.g., CI/CD, automated documentation generation, version control) for OpenADMET activities, with a focus on automation and efficiency.</li><li>Coordinate with OpenADMET team members to extract, process, and integrate data from public databases and FAIR repositories.</li><li>Implement strategies to store data in performant and reliable data warehouse/lake</li><li>Manage public releases of data into industry standard public databases (e.g, ChEMBL, PubChem, Therapeutic Data Commons)</li><li>Develop best practice SOWs for automated data workflows, including ingestion, preparation, and deposition into FAIR repositories, in collaboration with stakeholders.
<em>Collaboration and Dissemination</em></li><li>Engage with cross-functional and cross-institutional teams to align data workflows with machine learning and data analysis requirements.</li><li>Assist with onboarding and training team members in data curation and workflow management best practices.</li><li>Support dissemination of data to public repositories and other channels to maximize impact and usability.</li></ul><p><strong>Qualifications</strong></p><ul><li>Advanced degree in a relevant field such as computational chemistry, data science, bioinformatics, or a related discipline (PhD preferred, but candidates with a Masterâ€™s degree and significant experience are welcome).</li><li>Proven experience with data curation, validation, and dissemination practices, ideally within scientific or ADMET-focused datasets.</li><li>Strong understanding of data workflows and pipeline optimization for scientific research applications, particularly with experience in automated data ingestion and FAIR data principles.</li><li>Familiarity with remote work environments and tools (Google Workspace, Slack, GitHub, etc.), and experience working with distributed teams.</li><li>Excellent communication skills and the ability to work across interdisciplinary teams to address complex data challenges.
<em>Preferred Qualifications</em></li><li>Technical expertise in Python, SQL (and its variants), containerization (e.g., Docker, Kubernetes), and Git</li><li>Experience with cloud computing (e.g., AWS, Azure).</li><li>Experience with molecular data processing and modeling tools, such as RDKit, DeepChem, or similar.</li><li>Experience developing and managing FAIR data repositories or similar data-sharing infrastructures.</li><li>Background in data quality assurance, especially in ensuring accuracy for machine learning training and benchmarking.</li><li>Previous involvement in open-source, collaborative, and remote scientific projects.</li><li>Comfortable working in a distributed global team</li></ul><p><strong>Compensation</strong>
This is a full-time, fixed contract position with an expected gross salary range of $100,000-$125,000, depending on qualifications, experience, and location. OMSF offers standard benefits, including healthcare, retirement contributions, paid time off, and other employer contributions per local regulations. Salary will be negotiated in the local currency and may vary by location due to differences in mandatory employer contributions.</p><p><strong>How to Apply</strong>
Please apply using this <a href=https://forms.gle/g2uWWzx86a9xAYWF8><strong>form</strong></a>.</p><div class=header-two><img src=/OMSF/images/H2before.png alt><h2 id=omsf-expression-of-interest>OMSF Expression of Interest</h2></div><p>We are always looking for enthusiastic candidates to contribute to open source projects in molecular sciences, even if we don't have any available openings. If you are interested in building your career as a software scientist, project or product manager, documentation writer, etc, please fill out this <a href=https://forms.gle/ChzBhUWKBqVPk1QR7><strong>form</strong></a> or reach out to <a href=mailto:careers@omsf.io>careers@omsf.io</a>.</p></article></main><footer class=footer-blue><div class="column-1 column"><img class=logo-footer src=/images/logo-white.png><p>Unless otherwise noted, all content is licensed under the <a style=text-decoration:underline href=https://creativecommons.org/licenses/by/4.0/>CC BY 4.0 license.</a></p></div><div class="column-2 column"><a href=/about/mission>About</a><br><a href=/services/fiscal-sponsorship/>Services</a><br><a href=/projects/host-project>Projects</a><br><a href=https://news.omsf.io>News</a><br><a href=/support/how-to>Support</a><br></div><div class="column-3 column"><a href=https://twitter.com/openmsf><img class=logo-footer src=/images/twitter-icon.png></a>
<a href=mailto:info@omsf.io><img class=logo-footer src=/images/mail-icon.png></a>
<a href=https://www.linkedin.com/company/><img class=logo-footer src=/images/linkedin-icon.png></a></div><div class="column-4 column"><a href=/privacy-policy>Privacy Policy</a><br><a href=/bylaws>Bylaws</a><br><a href=/good-standing-policy>Good Standing Policy</a><br><a href=/antitrust-policy>Antitrust Policy</a></div></footer></div></body></html>